src/stores/SidePanelDocs/Store.ts

```diff
@@ -34,23 +33,13 @@ const getInitialState = (
         );
     },
 
-    setShow: (val) => {
-        set(
-            produce((state: SidePanelDocsState) => {
-                state.show = val;
-            }),
-            false,
-            'Side Panel Docs Show Updated'
-        );
-    },
-
     setUrl: (val) => {
         set(
             produce((state: SidePanelDocsState) => {
                 const urlIsFromEstuary = val.includes(iframeStringInclude);
 
                 state.url = val;
-                state.show = true;
+                // state.show = true;
```

Comment:
We should be good to just remove this totally.

Reply:
This is obviously a commented line of code that was simply missed in a cleanup effort.